web-components/src/components/table/ActionsTable.tsx

```diff
@@ -22,27 +22,23 @@ import {
 // } from 'web-components/lib/components/table/sort';
 import { TablePagination } from './TablePagination';
 
-const useStyles = makeStyles(theme => ({
-  borderLeft: {
-    // absolutely position border to get around MUI style quirks
-    position: 'absolute',
-    top: 0,
-    left: 0,
-    bottom: 0,
-    borderLeft: `1px solid ${theme.palette.info.light}`,
-  },
-  greyText: {
-    color: theme.palette.info.main,
-  },
+const BorderLeft = styled('div')(({ theme }) => ({
```

Comment:
As I mentioned in [Victor's PR](https://github.com/regen-network/regen-web/pull/856), I think we should try and gracefully deprecate our use of `makeStyles` to be better positioned for React 18. This is one example

Reply:
Sounds good.